pkg/codegen/hcl2/model/type_list.go

```diff
@@ -104,6 +104,10 @@ func (t *ListType) conversionFrom(src Type, unifying bool) ConversionKind {
 	})
 }
 
+func (t *ListType) GetAnnotations() []interface{} {
+	return nil
```

Comment:
Is this always going to be `nil`, or do we want to include a TODO, or placeholder message of some sort?

Reply:
This is going to be nil indefinitely. I'll file an issue for a proper design here; getting the type system to track annotations in a way that makes sense is tricky.

Reply:
https://github.com/pulumi/pulumi/issues/4466